site/docs/about/roadmap.mdx

```diff
@@ -25,12 +25,9 @@ For previously released features, see changelogs in the [What is new section](/a
 ~~- Component: Time input field~~   
 ~~- Component: Number input field~~   
 ~~- Icons: Next set of icons~~   
-
-
-
-- Component: Dialog
+~~- Component: Dialog~~
```

Comment:
Seems that the line change here will break in the actual view. Three white spaces in the end of the line does the trick in markdown. ( I tremble with the same problem every time)

Reply:
I can fix this.

Reply:
✅

Reply:
My editor did not like the three spaces 😬  I replaced them with good ol' <br/> - tag 😁